dev-utils/watch_ddb_stream.py

```diff
@@ -14,15 +14,15 @@
 DYNAMODB_STREAMS = make_dynamodb_stream_images_multicast()
 
 
-def make_accept_stream_images(item_slicer: Callable[[dict], str]):
+def make_accept_stream_images(item_slicer: Callable[[ItemImages], str]):
```

Comment:
Is this right? It seems to indicate that `item_slicer` will return a `str`, but the implementation of `make_item_slicer` below seems to return something that will return a `dict`?